snakemake/executors/ga4gh_tes.py

```diff
@@ -7,6 +7,7 @@
 import os
 import stat
 import time
+import math
```

Comment:
Seems like the imports are sorted alphabetically, with whole library imports preceding partial imports. So perhaps move this up between `asyncio` and `io`.

Reply:
done